webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -3489,30 +3485,13 @@ object RudderConfigInit {
     lazy val updatesEntryJdbcRepository = new LastProcessedReportRepositoryImpl(doobie)
 
     lazy val executionService = {
-      val maxCatchupTime  = {
-        val temp = FiniteDuration(RUDDER_REPORTS_EXECUTION_MAX_DAYS.toLong, "day") + FiniteDuration(
-          RUDDER_REPORTS_EXECUTION_MAX_MINUTES.toLong,
-          "minutes"
-        )
-        if (temp.toMillis == 0) {
-          logger.error(
-            "'rudder.aggregateReports.maxDays' and 'rudder.aggregateReports.maxMinutes' properties are both 0 or empty. Set using 30 minutes as default value, please check /opt/rudder/etc/rudder-web.properties"
-          )
-          FiniteDuration(30, "minutes")
-        } else {
-          temp
-        }
-      }
-      val maxCatchupBatch = FiniteDuration(RUDDER_REPORTS_EXECUTION_MAX_SIZE.toLong, "minutes")
-
```

Comment:
`RUDDER_REPORTS_EXECUTION_MAX_DAYS`, `RUDDER_REPORTS_EXECUTION_MAX_MINUTES`, `RUDDER_REPORTS_EXECUTION_MAX_SIZE` all seem to be unused now, they should also be removed from `configuration.properties.sample`